src/DynamoCoreWpf/ViewModels/Watch3D/AttachedProperties.cs

```diff
@@ -36,13 +36,7 @@ private static void ShowSelectedPropertyChanged(DependencyObject obj, Dependency
             if(obj is GeometryModel3D && obj.GetType() != typeof(BillboardTextModel3D))
             {
                 var geom = (GeometryModel3D)obj;
-                
-                if (geom.IsAttached)
-                {
-                    var host = geom.RenderHost;
-                    geom.Detach();
-                    geom.Attach(host);
-                }
+                // TODO DYN-973: Need new mechanism to trigger render update after selected/frozen/isolated properties change
```

Comment:
With Detach/Attach no longer possible (nor desirable, for performance reasons!) it's not clear what the most idiomatic way to force an update on these scene objects would be in our environment. In raw Helix-WPF, the idiomatic thing to do would be to use WPF dependency property binding to bind the material to the selected/frozen properties with a ValueConverter, but that doesn't look like the best option here.

Reply:
@Dewb - doing my own little investigation, seems that Detach is still possible - we need to override `OnDetach` now and call base.Detach from there for some reason.

see:
https://github.com/helix-toolkit/helix-toolkit/pull/404

Reply:
responded to this comment below, not sure why that comment about detach is not inline - BUT
in regards to freeze, select etc. Why don't we just use different materials?

Reply:
The `OnDetach` methods are overridable in base classes, but the Detach methods that trigger them have moved from their original publicly accessible location. Still investigating what the intended use it here.

Yes, swapping materials would be an example of what I meant by reimplementing features like freeze/selected appearances outside the shader pipeline.

Reply:
Here are my conclusions after some investigation

1. As we previously discussed Attach and Detach are no longer available publicly. They are now invoked internally whenever a model is being added or removed from Viewport3DX.Items
2. Hence, if you want a model to not be showed, it needed to be removed from Viewport3DX.Items
3. If a model is already been added to Viewport3DX.Items. You can still modify its properties at run-time
4. The only slight exception to 3 is the Geometry property. If you modify vertex, indices, colors you must explicitly call Geometry.UpdateVertices() or UpdateTriangles() etc... afterwards
5. I think the simplest way to highlight (or freeze) selected geometries is to modify the Color property of the model (The default shader multiplies the model's Color with each vertex color to get the final rendered color)